tests/FSharpx.Collections.Experimental.Tests/AaTreeTest.fs

```diff
@@ -0,0 +1,129 @@
+namespace FSharpx.Collections.Experimental.Tests
+
+open FSharpx.Collections
+open FSharpx.Collections.Experimental
+open Expecto
+open Expecto.Flip
+
+module AaTreeTest =
+    [<Tests>]
+    let testAaTree = 
+        testList "AaTree" [
+
+        (* Existence tests. *)
+        test "test isEmpty" {
```

Comment:
Can we divide the tests to have test per assertion/aspect and explain in the test name which aspect of `isEmpty` we are testing? Here I would go with something along the lines of `"test isEmpty returns true for an empty aatree"` and ` "test isEmpty returns false for a single element aatree"`. This is additional work for you to do now, but it will make future maintenance easier.